lib/modules/versioning/lambda-node/schedule.ts

```diff
@@ -0,0 +1,30 @@
+import dataFiles from '../../../data-files.generated';
+import { isStable } from '../node';
+
+interface LambdaSchedule {
+  cycle: string;
+  releaseLabel: string;
+
+  /**
+   * Either `true` if currently in support or a string indicating the date at which support will end
+   */
+  support: true | string;
+}
+
+export type LambdaData = Record<string, LambdaSchedule>;
+
+const lambdaSchedule: LambdaData = JSON.parse(
+  dataFiles.get('data/lambda-node-js-schedule.json')!,
+);
+
+export function findLambdaScheduleForVersion(
+  version: string,
+): LambdaSchedule | null {
+  const majorVersionMatch = /^v?([0-9]+)\./.exec(version);
```

Comment:
```suggestion
  const majorVersionMatch = regEx(/^v?([0-9]+)\./).exec(version);
```
move below condition (perf)